stores/hostdb.go

```diff
@@ -80,6 +80,44 @@ type (
 		Blocklist []dbBlocklistEntry `gorm:"many2many:host_blocklist_entry_hosts;constraint:OnDelete:CASCADE"`
 	}
 
+	// dbHostInfo contains information about a host that is collected and used
+	// by the autopilot.
+	dbHostInfo struct {
```

Comment:
I don't think we need the `gorm` tags anymore. At least not the ones for indices.